### ISSUE TITLE
Duplicate post on different page bug

### ISSUE THREAD

**Issue by akinsey** [epochtalk/epochtalk#515]
"I can see a post of mine on two pages of the paging of a given thread. If I change the setting to allow 50 post per page (instead of 25), then the post only shows once as it should. The post is last on the first page of the thread (post number 25 - if there were a number), and second on the following page (post number 27 - if there were a number)."

**Comment by akinsey** [comment]
@DdmrDdmrAlt2 do you have any more info on this bug? I can't seem to recreated it. I created a new thread with posts counting from 1 to 35. I tried a combination of hiding/purging posts in the middle and at the end of the page to try to recreate what you were talking about. 

Any idea of what actions were taken on the thread that might have resulted in the post ordering getting messed up? If not no worries, I will keep looking into this.

**Comment by DdmrDdmrAlt2** [comment]
I can't seem to recreate it now. It seemed more of a cahe thing, since the post was not repeated itself, but showed on two different consecutive pages of the thread. I've tried it on a fresh thread but no luck. Not sure how It showed-up originally.

What I have just noticed now is that you cannot hide (“delete”) the first post on each page of the same thread. I’ve just created a thread with 28 posts, set my setting to view 10 per page, and posts number 1, 11, and 21 are non-hidable (i.e. action icon not present).
